<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="A-PC" ignored="0" name="Test_All(failed)" tests="3" failures="2" timestamp="24 Jun 2022 09:57:15 GMT" time="73.756" errors="0">
  <testcase name="loginTestUsingDP" time="8.177" classname="com.mystore.testcases.LoginPageTest">
    <failure type="java.lang.AssertionError" message="expected [http://automationpractice.com/index.php?controller=my-account] but found [http://automationpractice.com/index.php?controller=authentication]">
      <![CDATA[java.lang.AssertionError: expected [http://automationpractice.com/index.php?controller=my-account] but found [http://automationpractice.com/index.php?controller=authentication]
at com.mystore.testcases.LoginPageTest.loginTestUsingDP(LoginPageTest.java:55)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginTestUsingDP -->
  <testcase name="loginTestUsingDP" time="7.856" classname="com.mystore.testcases.LoginPageTest">
    <failure type="java.lang.AssertionError" message="expected [http://automationpractice.com/index.php?controller=my-account] but found [http://automationpractice.com/index.php?controller=authentication]">
      <![CDATA[java.lang.AssertionError: expected [http://automationpractice.com/index.php?controller=my-account] but found [http://automationpractice.com/index.php?controller=authentication]
at com.mystore.testcases.LoginPageTest.loginTestUsingDP(LoginPageTest.java:55)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginTestUsingDP -->
  <testcase name="loginTestUsingPropertiesFile" time="10.412" classname="com.mystore.testcases.LoginPageTest"/>
</testsuite> <!-- Test_All(failed) -->
